shell/browser/api/electron_api_web_contents.cc

```diff
@@ -2492,6 +2492,58 @@ content::NavigationEntry* WebContents::GetNavigationEntryAtIndex(
   return web_contents()->GetController().GetEntryAtIndex(index);
 }
 
+bool WebContents::DeleteNavigationEntryAtIndex(int index) {
+  if (index < 0 || index >= GetHistoryLength())
```

Comment:
You can use `CanGoToIndex`

Reply:
Ah I see, thanks for the suggestion. Changed.